dashboard/test/ui/features/projectsLegacy.feature

```diff
@@ -0,0 +1,59 @@
+# TODO (brent): disabling this feature to start with, as it takes a while to run,
```

Comment:
My reasoning for keeping it around was so that it would be easy to add it to a long running suite if desired, or if we wanted to run it as a one off. At this point I think our design has maybe changed enough that it's not even worth that. I'll remove.